bvt/op-ci-basic.xml

```diff
@@ -96,5 +97,17 @@
             </testcase>
         </test>
 
+        <test>
+            <testcase>
+                <cmd>op-ci-bmc-run "op_ci_bmc.validate_lpar()"</cmd>
+            </testcase>
+        </test>
+
+        <test>
+            <testcase>
+                <cmd>op-ci-bmc-run "op_ci_bmc.outofband_fwandpnor_update_hpm()"</cmd>
```

Comment:
We'd like to test the image we're updating so please do the update, and then the IPL, lpar validation, SEL checking, ...